src/ZombieDriver.php

```diff
@@ -125,7 +125,10 @@ public function visit($url)
   }
 });
 JS;
-        $this->server->evalJS($js);
+        $out = $this->server->evalJS($js);
```

Comment:
1. Are you sure, that JavaScript errors (if any) will be returned as method call result?
2. Does same error catching technique apply to other js-enabled drivers, like Selenium?

Reply:
@aik099 visiting a URL in Selenium uses a native Selenium method. So we don't need to implement error handling (Selenium does).

Reply:
@aik099 see the JS code just above. On error, it outputs some stuff to the stream.

Reply:
The zombie "browser" object's method "visit" has a callback as its second argument that will be called if an error occurs. Currently this error information is just thrown away. I found it very helpful to have this error information in my project when making behat tests.